src/components/Header/index.tsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+
+import { Link } from '..';
+import logoLight from '../../images/logo-light.svg';
+import logoDark from '../../images/logo-dark.svg';
+import styles from './header.module.scss';
+
+// should be reaplaced with GraphQL data
+
+const content = {
+  navigationLinks: [
+    {
+      link: 'https://nodejs.org/en/docs/',
+      text: 'Documentation',
+    },
+    {
+      link: 'https://github.com/nodejs/nodejs.dev',
+      text: 'GitHub',
+    },
+    {
+      link: 'https://nodejs.org/en/download/',
+      text: 'Downloads',
+    },
+  ],
+  toggleTheme: 'Toggle theme',
+};
+
+const Header = (): JSX.Element => (
+  <header className={styles.mainHeader}>
+    <nav className={styles.navigation}>
+      <Link to="/">
+        <img
+          src={logoLight}
+          alt="Node.js"
+          className={[styles.navigationLogo, 'light-mode-only'].join(' ')}
+        />
+        <img
+          src={logoDark}
+          alt="Node.js"
+          className={[styles.navigationLogo, 'dark-mode-only'].join(' ')}
+        />
+      </Link>
+      {content.navigationLinks.map(
+        (item, i): JSX.Element => (
+          <Link
```

Comment:
We should use the Link from Gatsby to toggle the active state:
`<Link to="/learn" activeStyle={activeStyleTab} partiallyActive>`

Reply:
``` javascript
const activeStyleTab = {
  fontWeight: 'var(--font-weight-semibold)',
  color: 'var(--color-text-accent)',
  borderBottom: 'var(--space-04) inset var(--color-text-accent)',
};
```